pastepwn/analyzers/md5hashanalyzer.py

```diff
@@ -7,5 +7,5 @@ class MD5HashAnalyzer(RegexAnalyzer):
     name = "MD5HashAnalyzer"
 
     def __init__(self, actions):
-        regex = r"[a-f0-9]{32}"
+        regex = r"\b[a-f0-9]{32}\b"
```

Comment:
👍

Reply:
Sorry about them being in the same PR. I'm still learning how this stuff works. I uploaded the Adobe PR and then started working on the MD5 issue. When I pushed it via Git Desktop it added it to the same PR. I'm assuming there's a way to separate them but I don't know how unfortunately.

Reply:
Ah no worries :) 

Maybe for the future: When you create a Pull request you should create a new **branch** in your fork. Because everything you will commit to one branch will be part of the PR.

![grafik](https://user-images.githubusercontent.com/5798157/66333713-d525ad00-e937-11e9-9034-55586b785f62.png)

Because with git you don't want to merge commits. You want to merge branches. And each feature should be developed in its own branch. If you have further questions, feel free to ask 😄